src/govuk/components/tabs/tabs.mjs

```diff
@@ -199,10 +198,15 @@ Tabs.prototype.showTab = function ($tab) {
  * Get tab link by hash
  *
  * @param {string} hash - Hash fragment including #
- * @returns {HTMLAnchorElement | null} Tab link
+ * @returns {HTMLAnchorElement | undefined} Tab link
```

Comment:
@36degrees Yeah I think we agreed we didn't want a double "null or undefined" return type:

```
/**
 * @returns {HTMLAnchorElement | undefined | null} Tab link
 */
```

Let me know which one should go where though, I'll make it happen 👍

Reply:
Thought: do we need to be pedantic about whether it's null or undefined? Could we have a `Nullish` type that matches both and that we put when we expect the result to be null or undefined?

I'm worried about spending time typing things rather than getting things done?

Reply:
@romaricpascal Yeah course all I'm after is the JSDoc types linting correctly

To avoid reflowing our code just match the JSDoc to what the code _currently_ does?

So early returns will be `undefined` but `document.querySelector()` can be `null`

```js
/**
 * @returns {HTMLAnchorElement | undefined | null} Tab link
 */
```

☝️ I didn't think we'd like that hence avoiding the early `undefined` return

Reply:
Done 👍